<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-2KN8RMI" name="Default test(failed)" tests="2" failures="1" timestamp="24 Jul 2018 10:55:42 GMT" time="0.02" errors="0">
  <testcase name="m2" time="0.003" classname="com.scp.maven.TestNg.ATest">
    <failure type="java.lang.AssertionError" message="This test case is failed...!">
      <![CDATA[java.lang.AssertionError: This test case is failed...!
	at com.scp.maven.TestNg.ATest.m2(ATest.java:29)
... Removed 25 stack frames]]>
    </failure>
  </testcase> <!-- m2 -->
  <testcase name="m1" time="0.0" classname="com.scp.maven.TestNg.ATest">
    <skipped/>
  </testcase> <!-- m1 -->
</testsuite> <!-- Default test(failed) -->
